CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/Services/ApiClient.cs

```diff
@@ -79,5 +79,41 @@ public async Task<HttpResponseMessage> PostAsync(string resource, string body)
                 throw;
             }
         }
+        public async Task<HttpResponseMessage> PutAsync(string resource, string body)
+        {
+            try
```

Comment:
No need for **try-catch** we won't do anything with an error in this method, all callers should take care of an exception themselves